<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>13</y>
      <width>801</width>
      <height>501</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="Add">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>50</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Delete">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>137</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAddData">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>13</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDelete">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>140</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>13</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAddKey">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>13</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSearch">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>260</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>13</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="Search">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>260</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSearch">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>310</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Task">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>190</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Task</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>10</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter data to add:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>55</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter key to add:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>116</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter key to delete:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>235</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter key to search:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>290</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>The results of search:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
